python/test/unit/language/test_core_amd.py

```diff
@@ -955,7 +955,7 @@ def convert_float_to_float32(fp: torch.tensor, dtype=None):
 
     extended_exp = ((1 << (tl.float32.primitive_bitwidth - tl.float32.fp_mantissa_width - 1)) - 1) << tl.float32.fp_mantissa_width
     # special cases, exp is 0b11..1
-    if dtype in [tl.float8e4nv, tl.float8e4b15]:
+    if dtype in [tl.float8e4nv, tl.float8e4b15, tl.float8e4b8, tl.float8e5b16]:
```

Comment:
For MI300 fp8 types, this does not apply, as 0x0111111 and 0x1111111 are not NaNs. Instead, 0x10000000 is inf/nan.

Reply:
Thanks. fixed.

Reply:
Hi @wenchenvincent, do you have other comments for this PR? 
If not, could you please help approve it? Thanks